.github/workflows/integration_tests.yml

```diff
@@ -40,11 +40,11 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-buildx-${{ matrix.target }}-
     - name: Build docker images
-      #env:
-      #  DOCKER_TRACE: 1
       run: |
         docker buildx create --driver docker-container --use
         bin/docker-build-${{ matrix.target }}
+        # Ensure all of the images are tagged under cr.l5d.io so pulls work
+        DOCKER_REGISTRY=cr.l5d.io bin/docker-build-${{ matrix.target }}
```

Comment:
This and the change below can be discarded now that `DOCKER_REGISTRY` defaults to `cr.l5d.io`

Reply:
The GitHub secret value is unchanged, since we still publish directly to ghcr...

Reply:
I think you wanna override `DOCKER_REGISTRY` to `ghcr.io/linkerd` in  `release.yml` which is where we actually push images.